.travis.yml

```diff
@@ -70,7 +72,7 @@ env:
 - pacifica-metadata-cmd dbsync
 - pacifica-uniqueid-cmd dbsync
 - pacifica-metadata & echo $! > metadata.pid
-- pacifica-archiveinterface & echo $! > archivei.pid
+- uwsgi --http-socket :9080 --master --module 'pacifica.archiveinterface.wsgi' & echo $! > archivei.pid
```

Comment:
@karcaw I've converted the archiveinterface to uwsgi.

Reply:
nice